gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,173 @@
+              type <NxCode>string</NxCode>, which is the message to be displayed on the toast.
+              For example: <NxCode>showSuccessToast('Account successfully created.');</NxCode>
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>showWarningToast</NxCode></NxTable.Cell>
+            <NxTable.Cell>function</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              Function to call in order to display a warning toast. The function takes in a parameter of
+              type <NxCode>string</NxCode>, which is the message to be displayed on the toast.
+              For example: <NxCode>showWarningToast('Unauthorized access may be monitored.');</NxCode>
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>showInfoToast</NxCode></NxTable.Cell>
+            <NxTable.Cell>function</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              Function to call in order to display an info toast. The function takes in a parameter of
+              type <NxCode>string</NxCode>, which is the message to be displayed on the toast.
+              For example: <NxCode>showInfoToast('Check permissions.');</NxCode>
+            </NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Body>
+      </NxTable>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Simple Toast Example"
+                        id="nx-toast-example"
+                        liveExample={NxToastExample}
+                        codeExamples={nxToastExampleCode}>
+      An example of a simple toast.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Toast Variations Example"
+                        id="nx-toast-variations-example"
+                        liveExample={NxToastVariationsExample}
+                        codeExamples={nxToastVariationsExampleCode}>
+      An example displaying all the variations of toasts.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Multiple Toasts Example"
+                        id="nx-toast-multiple-example"
+                        liveExample={NxToastMultipleExample}
+                        codeExamples={nxToastMultipleExampleCode}>
+      An example of displaying multiple toasts at once. This example also shows how a toast
+      appears and is stacked on top of other toasts.
+    </GalleryExampleTile>
+
+    <NxTile>
+      <NxTile.Header>
+        <NxTile.HeaderTitle>
+          <NxH2>Toasts Used With NxGlobalHeader Example</NxH2>
+        </NxTile.HeaderTitle>
+      </NxTile.Header>
+      <NxTile.Content>
+        <NxP>
+          <NxTextLink href="#/NxGlobalHeaderToastExample">
+            Click here to navigate to the live example.
+          </NxTextLink>
+        </NxP>
+        <CodeExample content={nxGlobalHeaderToastExampleCode} />
```

Comment:
Missing the accordion

Reply:
Added in the accordian here: [91c0683](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/91c068321f25b891bf2f411e586fa7a6101eeef3)

Reply:
> Missing the accordion

Added in the accordian here: [91c0683](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/91c068321f25b891bf2f411e586fa7a6101eeef3)